graphicsmagick/src/wand/drawing.rs

```diff
@@ -152,9 +152,8 @@ impl DrawingWand {
     ///
     /// must be deallocated by the user when it is no longer needed.
     ///
-    pub fn get_clip_path(&self) -> Result<String, Utf8Error> {
-        let c = unsafe { MagickDrawGetClipPath(self.wand) };
-        c_str_to_string(c)
+    pub fn get_clip_path(&self) -> Option<MagickCString> {
+        unsafe { MagickCString::new(MagickDrawGetClipPath(self.wand)) }
```

Comment:
@jmjoy Is it possible for these APIs that returns a c-style string to return `NULL`?

Reply:
From the CI failure, it seems that this is absolutely possible.